src/api/helpers/get-inactive-applications/index.ts

```diff
@@ -15,15 +14,13 @@ const getInactiveApplications = async (context: Context): Promise<Application[]>
   try {
     console.info('Getting inactive applications - getInactiveApplications helper');
 
-    const thirtyDaysLimit = getThirtyDaysBeforeNow();
-
     /**
      * queries in progress applications which are older than 30 days from the db
      * returns array of id and status
      */
     const applications = (await context.query.Application.findMany({
       where: {
-        AND: [{ status: { in: [IN_PROGRESS] } }, { updatedAt: { lt: thirtyDaysLimit } }],
+        AND: [{ status: { in: [IN_PROGRESS] } }, { submissionDeadline: { lt: new Date() } }],
```

Comment:
Can we update the documentation to e.g

```ts
/**
  * queries "in progress"/${IN_PROGRESS} applications which have a submissionDeadline that are....
  * returns array of ids and statuses
  */
```

Reply:
The "elapsed" part needs more of a description 😛 

Do we need the `getThirtyDaysBeforeNow` helper anymore?

Reply:
Nice but sorry the documentation isn't clear, can we mention `IN_PROGRESS` status with all caps, also "has is in the ..." ? i think something like this would be nice:

```ts
/**
 * Queries applications that:
 * - have a status of IN_PROGRESS
 * - have a submission deadline in the past
 */
```